spec/models/user_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  first_user = User.create(name: 'Johnie', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Tech enthusiast.',
+                           posts_counter: 0)
+
```

Comment:
- [ ] Kindly include some positive test case also. for ex;- 
`first_user.name` to equal `'Johnie'`